server/src/main/java/com/google/sps/utils/ReportReader.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Description: File for reading in rows of CSV data
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+
+import com.google.sps.utils.Ad;
+import com.google.sps.utils.AdRowProcessor;
+import java.io.*;
```

Comment:
Java guideline 3.3.1 - No wildcard imports. You must list out explicitly what subclasses you're using in this file

Reply:
Got it